examples/llm_inference/android/README.md

```diff
@@ -0,0 +1,43 @@
+# MediaPipe LLM Inference Android Demo
+
+### Overview
+
+This is a sample app that demonstrates how to use the LLM Inference API to run common text-to-text generation tasks like information retrieval, email drafting, and document summarization.
+
+This application must be run on a physical Android device to take advantage of the device GPU.
+
+![LLM Inference Demo](llm_inference.png)
+
+## Build the demo using Android Studio
+
+### Prerequisites
+
+*   The **[Android Studio](https://developer.android.com/studio/index.html)**
+    IDE. This sample has been tested on Android Studio Hedgehog.
+
+*   A physical Android device with a minimum OS version of SDK 24 (Android 7.0 -
+    Nougat) with developer mode enabled.
+
+### Building
+
+*   Open Android Studio. From the Welcome screen, select Open an existing
+    Android Studio project.
+
+*   From the Open File or Project window that appears, navigate to and select
+    the mediapipe/examples/llm_inference/android directory. Click OK. You may
+    be asked if you trust the project. Select Trust.
+
+*   If it asks you to do a Gradle Sync, click OK.
+
+*   With your Android device connected to your computer and developer mode
+    enabled, click on the green Run arrow in Android Studio.
+
+### Models used
+
+[//]: # (TODO: Update this section with proper instructions on how to download a model)
+You can download one of the models available on [TBD](https://ai.google.dev/tbd).
```

Comment:
Are we able to host the model (like on ai.google.dev)? If so, we should add a download script instead so the sample is very plug-and-play.

Reply:
I don't think we'll be hosting the model. 

@schmidt-sebastian any suggestions on how we should reference the process of obtaining a model in this README ?